### ISSUE TITLE
RA and Dec overhead

### ISSUE THREAD

**Issue by techiesoft** [skyfielders/python-skyfield#310]
Hello,
For a given RA(Right Ascension) and Declination, how can i find where that is located on the earth as a subpoint. Pls give me an example for the same.

**Comment by techiesoft** [comment]
Any help on this would be helpful. Thanks

**Comment by techiesoft** [comment]
is my question correct or is it tough to achieve this?

**Comment by techiesoft** [comment]
@brandon-rhodes can you pls help me on this

**Comment by barrycarter** [comment]
My thinking would be that the overhead right ascension would be the local sidereal time and the overhead declination would the latitude. I'll let Brandon confirm here. If what I'm saying is correct, we probably don't need a special new function to compute this.

**Comment by brandon-rhodes** [comment]
Your question raises a bigger question that I'm not sure how we'll address going forward: given a library with features that can be combined in many ways, does the documentation eventually need to expand to show all the ways they can be combined, or is there a way the documentation of the individual pieces can be so clearly explained that the parts can be put together independently by users?

I created the Examples page recently to absorb "how to put things together" questions. I'm not going to worry right now over whether, if only `position_from_radec()` and `subpoint()` had been better and more brilliantly documented, it would be clear that `position_from_radec().subpoint()` is the solution here. Instead, I've just added it as an example:

https://rhodesmill.org/skyfield/examples.html#what-latitude-and-longitude-is-beneath-this-right-ascension-and-declination

But, just in case: if anyone reading _does_ see shortcomings in the documentation of `position_from_radec()` and `subpoint()` that could have led users to not imagine combining them, let me know and I'll try to get them fixed!
